skeleton/default.nix

```diff
@@ -0,0 +1,60 @@
+{ system ? builtins.currentSystem
+, crossSystem ? null
+, config ? {}
+# Import IOHK common nix lib
+, iohkLib ? import ./nix/iohk-common.nix { inherit system crossSystem config; }
+# Use nixpkgs pin from iohkLib
+, pkgs ? iohkLib.pkgs
+}:
+
+with import ./nix/util.nix { inherit pkgs; };
+
+let
+  haskell = iohkLib.nix-tools.haskell { inherit pkgs; };
+  src = iohkLib.cleanSourceHaskell ./.;
+
+  # Example of using a package from iohk-nix
+  # TODO: Declare packages required by the build.
+  inherit (iohkLib.rust-packages.pkgs) jormungandr;
+
+  # Import the Haskell package set.
+  haskellPackages = import ./nix/default.nix {
+    inherit pkgs haskell src;
+    # Pass in any extra programs necessary for the build as function arguments.
+    # TODO: Declare packages required by the build
+    inherit jormungandr;
+    inherit (pkgs) cowsay;
+    # Provide cross-compiling secret sauce
+    inherit (iohkLib.nix-tools) iohk-extras iohk-module;
+  };
+
+in {
```

Comment:
Maybe add a bit of prose for this? Something like: "This defines the top-level attribute set available in `default.nix`. You can build attributes from here with `nix build -f default.nix <attr>`."

Reply:
Agreed :)